services/dy-3dvis/simcoreparaviewweb/src/input-retriever.py

```diff
@@ -18,6 +20,15 @@
 from simcore_sdk.nodeports.nodeports import PORTS
 
 _INPUT_PATH = Path(os.environ.get("PARAVIEW_INPUT_PATH"))
+
+def move_file(filename):
+    if local_path.exists():
```

Comment:
where is local_path defined? so you mean filename?

Reply:
weird that it actually could still run...
refactored this and now it looks more beautiful... hopefully